README.md

```diff
@@ -1 +1,127 @@
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Available Scripts](#available-scripts)
+  - [Author](#author)
+  - [Future Feature](#future-feature)
+  - [Contributing](#contributing)
+  - [Show your support ](#show-your-support-)
+  - [Acknowledgments](#acknowledgments)
+  - [License](#license)
+
+
+# Budget App
+
+
+**Budget App** is a web mobile friendly app that supports user registration and authentication using the Devise gem, a user can create his own account on the app and after logging in she/he will be able to add categories and then add transactions under those categories, the app provide some metrics like the total value of transactions under each category, the app was build using Ruby on Rails framework.
+
+## Built With
+
+### Tech Stack 
+
+<ul>
+  <li><a href="https://www.ruby-lang.org/en/news/2022/11/24/ruby-3-1-3-released/">Ruby 3.1</a></li>
+  <li><a href="https://rubyonrails.org/">Ruby on Rails 7</a></li>
+  <li><a href="https://www.postgresql.org/">PostgreSQL 15</a></li>
+</ul>
+
+### Key Features
+
+- **Register an account**
+- **Create categories and transactions**
+- **Metrics about user transactions**
+
+## Live Demo 
+
+- [Live Demo Link](https://sajeel-budget-app.onrender.com/)
+
+## Getting Started 
+
+> To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+  - <a href="https://www.ruby-lang.org/en/news/2022/11/24/ruby-3-1-3-released/">Ruby</a>
+  - <a href="https://rubyonrails.org/">Ruby on Rails</a>
+  - <a href="https://www.postgresql.org/">PostgreSQL</a>
+  - any code editor
+
+### Setup
+
+To setup the project follow the steps:
+
+1. Clone this project using Git Bash: 
+    ``` 
+    git clone git@github.com:Sajeelzafar/budget_app.git
+    ```
+  
+2. Inside the project directory, install the project's dependencies
+    ```
+    bundle install
+    ``` 
+
+3. Create the app's database
+    ```
+    rails db:create
+    ``` 
+
+4. Setup the app's database
+    ```
+    rails db:setup
+    ``` 
+
+
+### Available Scripts
+
+In the project directory, you can run:
+
+- ```
+  rails server
+  ```
+  Runs the app local server
+
+- ```
+  bundle exec rspec
+  ```
+  Launches the test runner.
+
+## Author
+
+👤 **Sajeel Zafar**
+
+- GitHub: [@Sajeelzafar](https://github.com/Sajeelzafar)
+- Twitter: [@SajeelZafar6](https://twitter.com/SajeelZafar6)
+- LinkedIn: [Sajeel Zafar](https://www.linkedin.com/in/sajeelzafar/)
+
+## Future Feature
+
+- **Create a user profile**
+- **Create a desktop version**
+
+## Contributing 
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support <a name="support"></a>
+
+If you like this project give it a star!
+
+## Acknowledgments 
+
+- Microverse ❤️ 
+
+- Credit goes to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) whose original design was used.
+
+## License 
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please kindly fix the broken link, the link that you provided in the README file is broken 🙃 🙃